lsst/ts/laser/csc.py

```diff
@@ -158,7 +191,7 @@ def assert_propagating(self, action):
         -------
 
         """
-        self.model.stop()
+        self.model._laser.clear_fault()
 
 
     async def do_setValue(self, id_data):
```

Comment:
Also not implemented? Maybe raise an exception then?

On a side note, do you actually need `do_enterControl`?